README.md

```diff
@@ -59,6 +59,13 @@ To add items to this page, simply send a pull request. ([contributing guide](CON
 * [Joint Speech Recognition and Speaker Diarization via Sequence Transduction](https://arxiv.org/abs/1907.05337), 2019
 * [Says who? Deep learning models for joint speech recognition, segmentation and diarization](https://ieeexplore.ieee.org/abstract/document/8462375), 2018
 
+#### Online speaker diarization
+
+* [Turn-to-Diarize: Online Speaker Diarization Constrained by Transformer Transducer Speaker Turn Detection](https://arxiv.org/abs/2109.11641), 2021
```

Comment:
Make sure each paper only appear once on this paper.

Reply:
Fixed.